src/controllers/schedule.js

```diff
@@ -0,0 +1,281 @@
+const express = require('express');
+const moment = require('moment-timezone');
+
+const ScheduleController = (userModel, taskModel, authService, googleAPIService, solve) => {
```

Comment:
Not sure this controller needs `solve` as a parameter

Reply:
Oh, haha. I didn't realize that was still there! I actually originally had `solve` in a separate file, which I undid before the commit.